src/jit/flowgraph.cpp

```diff
@@ -7036,7 +7036,10 @@ GenTreeCall* Compiler::fgGetStaticsCCtorHelper(CORINFO_CLASS_HANDLE cls, CorInfo
         callFlags |= GTF_EXCEPT;
     }
 
-    return gtNewHelperCallNode(helper, type, callFlags, argList);
+    GenTreeCall* result = gtNewHelperCallNode(helper, type, argList);
+    result->gtFlags |= callFlags;
```

Comment:
This line is not redundant since in this method callFlags can also have GTF_CALL_HOISTABLE. The setting of GTF_EXCEPT above is redundant and I will remove it.